components/pricing.tsx

```diff
@@ -0,0 +1,156 @@
+import React from "react";
+import {
+  Card,
+  CardContent,
+  CardFooter,
+  CardHeader,
+  CardTitle,
+} from "@/components/ui/card";
+import { Button } from "@/components/ui/button";
+import { Check, Download } from "lucide-react";
+import Link from "next/link";
+
+interface PricingTierProps {
+  title: string;
+  price: string;
+  description: string;
+  features?: string[];
+  buttonText?: string;
+  isFree?: boolean;
+}
+
+const PricingTier: React.FC<PricingTierProps> = ({
+  title,
+  price,
+  description,
+  features,
+  buttonText,
+  isFree = false,
+}) => (
+  // these colors shouldn't be hard coded (ex. border-[#4CC9A7]) so it needs to be fixed but I am just following figma for now
+  <Card className="flex h-full w-full flex-col border border-[#4CC9A7]">
```

Comment:
```suggestion
  <Card className="flex h-full w-full flex-col border border-primary-700">
```

let's use `border-primary-700`. it's the closest color to the color used in the figma, just a bit darker.